asb-native/src/main/java/org/ballerinax/asb/util/ModuleUtils.java

```diff
@@ -39,6 +40,18 @@ public static Module getModule() {
     }
 
     public static void setModule(Environment env) {
+        String logLevel = "";
+        try {
+            logLevel = System.getenv("ASB_CLOUD_LOGS");
+        } catch (Exception e) {
+            // If a security manager exists, its checkPermission method is called with a
+            // RuntimePermission("getenv."+name) permission. This may result in a
+            // SecurityException being thrown. If no exception is thrown the value of the
+            // variable name is returned.
+        }
+        if (logLevel == null || logLevel.isEmpty() || !logLevel.equalsIgnoreCase("ACTIVE")) {
+            LogManager.getLogManager().reset();
```

Comment:
When logLevel.equalsIgnoreCase("ACTIVE") also we need to reset?

Reply:
Yes, this is the way to stop azure SDK logs.